backend.py

```diff
@@ -67,7 +67,7 @@ class Direction(Enum):
     S = 180, (0, -1), 2
     W = 270, (-1, 0), 3
 
-    def __new__(cls, keycode, vector, tile_property):
+    def __new__(cls, degrees, coor_delta, tile_property):
```

Comment:
Co znamená to `coor_delta`?

Reply:
Zpusob zapisu jsem zapujcila z toho, jak jsi to vcera pojmenoval na tabuli. A proc delta? (1, 0) nejsou koordinaty v pravem slova smyslu, je to hodnota, o kterou (vynasobenou delkou pohybu) se "ty prave" koordinaty zmeni. Proto mi nazev coordinates nebo new_coordinates nepripada zde jako spravny, nevystihuje podstatu vyznamu teto dvojice.